library/pk.c

```diff
@@ -175,7 +184,6 @@ int mbedtls_pk_setup_rsa_alt( mbedtls_pk_context *ctx, void * key,
  */
 int mbedtls_pk_can_do( const mbedtls_pk_context *ctx, mbedtls_pk_type_t type )
 {
-    /* null or NONE context can't do anything */
```

Comment:
Why remove this comment (in "Implement parameter validation in pk, pkparse and pkwrite")? It helps to explain why `PK_VALIDATE` is not being used here.

Reply:
I found that comment unclear: what's “null or NONE” in this context? And once I understood that it meant “NULL or with type MBEDTLS_PK_NONE”, I found it uninformative: yes, I can see that it can't do anything, that's what it means to have can_do return false. I've now added a comment that explains _why_ can_do behaves this way.